aurora/adas_files.py

```diff
@@ -32,7 +32,7 @@
 )
 
 
-def get_adas_file_loc(filename, filetype="adf11"):
+def get_adas_file_loc(filename, filetype="adf11", tmp_folder=None):
```

Comment:
@odstrcilt why do you need this `tmp_folder` here?

Reply:
I have no idea. I will fix it.